go-controller/pkg/node/gateway.go

```diff
@@ -337,6 +337,7 @@ func gatewayInitInternal(nodeName, gwIntf, egressGatewayIntf string, gwNextHops
 	l3GwConfig := util.L3GatewayConfig{
 		Mode:           config.Gateway.Mode,
 		ChassisID:      chassisID,
+		BridgeID:       gatewayBridge.bridgeName,
```

Comment:
@tssurya This commit should be removed if we can take the bridge id from the config instead?